torch/lib/c10d/Store.hpp

```diff
@@ -30,6 +30,8 @@ class Store {
 
   virtual int64_t add(const std::string& key, int64_t value) = 0;
 
+  virtual void deleteKey(const std::string& key) = 0;
```

Comment:
Usually a delete API returns true/false about whether the key was removed or not. I think this should return a bool.